### ISSUE TITLE
Add a clear recent files on exit feature

### ISSUE THREAD

**Issue by RagglenLove** [tenacityteam/tenacity-legacy#445]
**Is your feature request related to a problem? Please describe.**

Clear recent files on exit is a feature that was most needed on Audacity, but for some reason was never worked on. Sometimes I would forget to clear recent files and I found that this was an issue of privacy since the next user of the computer can enter Audacity and see what files you have been working on.

**Describe the solution you'd like**

Add a clear recent files on exit feature.

**Describe alternatives you've considered**

**Additional context**

**Comment by petersampsonaudacity** [comment]
There is a simple workaround _(Audacity has had this for a long time)_:

![image](https://user-images.githubusercontent.com/50205984/128696544-fd110014-9873-42c3-b46f-f9ef4fe32f00.png)

**Comment by RagglenLove** [comment]
> There is a simple workaround _(Audacity has had this for a long time)_:
> 
> ![image](https://user-images.githubusercontent.com/50205984/128696544-fd110014-9873-42c3-b46f-f9ef4fe32f00.png)

Yes, I know this, but that is the manual way to clear recent files, sometimes, I would forget to do that sometimes, so what I'm proposing is an option is the settings to clear the recent files automatically when you exit Tenacity.
